### ISSUE TITLE
Error invoking kernel: CUDA error (700)

### ISSUE THREAD

**Issue by xzengae** [TinkerTools/Tinker-OpenMM#9]
Hi,
I've installed Tinker-OpenMM on our GPU clusters. But when I submit a bunch of jobs to run the simulations, some jobs could run successfully, some just throws an error after running several minutes or nearly 1 hour, even in the same node (seems that the job only runs in cpu because the simulation is super slow compared to running on GPU):

terminate called after throwing an instance of 'OpenMM::OpenMMException'
what(): Error invoking kernel: CUDA error (700)

The system is CentOS release 6.9,
GPU cards: Tesla K20m (some nodes are Tesla K20Xm), compute mode: default. 
CUDA version: 10.1 (I've also tried 8.0, still doesn't work).

Any one have any thoughts about this? Thanks!

**Comment by xzengae** [comment]
A follow up, I found this problem is related to restraints applied on the torsion angle, not related to the GPU stuff. When I remove the restraints, all the jobs could run successfully. However, if I add the restrain to the key file, the job cannot run, even only on CPU, just hang there, without any output. I've tested this on different clusters, including my local computer. The keyword I used to add the torsion angle restrain is "RESTRAIN-TORSION". Still don't know why this could happen.
